cli/cmd/install.go

```diff
@@ -316,12 +314,25 @@ func renderCRDs(w io.Writer) error {
 		return err
 	}
 
-	// The CRD chart does not take any value configuration.
-	values, err := l5dcharts.NewValues()
+	// Load defaults from values.yaml
+	valuesFile := &loader.BufferedFile{Name: l5dcharts.HelmChartDirCrds + "/values.yaml"}
+	if err := charts.ReadFile(static.Templates, "/", valuesFile); err != nil {
+		return err
+	}
+	var defaultValues map[string]interface{}
+	err := yaml.Unmarshal(charts.InsertVersion(valuesFile.Data), &defaultValues)
```

Comment:
linkerd-crds doesn't need to have `linkerdVersionValue` replaced in its `values.yaml`
```suggestion
	err := yaml.Unmarshal(valuesFile.Data, &defaultValues)
```